src/EncryptionSDK/Materials.dfy

```diff
@@ -9,7 +9,7 @@ module Materials {
 
   type EncryptionContext = seq<(seq<UInt8>, seq<UInt8>)>
 
-  datatype EncryptedDataKey = EncryptedDataKey(providerID : string, 
+  datatype EncryptedDataKey = EncryptedDataKey(providerID : string,
```

Comment:
Yes, if you are going to trim whitespace,
then I would not add a whole line of whitespace

Reply:
Removing trailing whitespace is only a minor option of the formatter that could be removed.

But the main goal of the formatter is not to add invisible spaces, it's to use space so that the code becomes more logical and more legible, in a way that is automated, so that a user do not need to bother to much about knowing "what is the right indentation". If they use the formatter, the formatter will have this opinion for them.